src/coffee/commands/images-convert.coffee

```diff
@@ -12,6 +12,7 @@ program
 .option '-d, --descriptions <path>', 'set image descriptions file path'
 .option '-r, --regex [name]', 'an optional RegExp used for filtering listed files (e.g.: /(.*)\.jpg/)', ''
 .option '-l, --logFile <path>', 'optionally log to a file instead of printing to console (errors will still be printed to stderr)'
+.option '-e, --expire <value>', 'set an expiration header - Cache-control: max-age. Default 30 days.', parseInt
```

Comment:
Please move configuration for compress/expire headers to the AWS S3 description yaml file